src/index.html

```diff
@@ -16,17 +16,33 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="search-bar__page">
+    <form
+      class="search-bar__form search-bar__form--big"
+      action="http://localhost:8080/"
+      data-qa="big"
+    >
+      <input
+        class="search-bar__input search-bar__input--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        autocomplete="on"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar__form search-bar__form--small"
+      action="http://localhost:8080/"
```

Comment:
Don't use spaces in links. The action URL 'http://localhost:8080/' should not contain a trailing slash if not necessary.